effort_controllers/test/descriptions.hpp

```diff
@@ -0,0 +1,174 @@
+// Copyright 2020 ros2_control Development Team
```

Comment:
We should take care of #133 to avoid copying this test file. (see ros-controls/ros2_control#247)